templates/request_page.hbs

```diff
@@ -128,6 +128,30 @@
         {{/if}}
 
         <dl class="request-details">
+          {{#with survey_response}}
+            <dt>{{t 'rating'}}</dt>
+            <dd>
+              <div>
+                {{#with rating}}
+                  {{#is scale_type 'numerical'}}
+                    {{t 'numerical_rating' rating=value max_value=max_value}}
+                  {{else}}
+                    {{scale_value}}
+                  {{/is}}
+                {{/with}}
+              </div>
+              <div>
+                <a href="{{page_path 'survey_response' survey_response_id=id}}" target="_blank">
+                  {{#if expired}}
+                    {{t 'view_rating'}}
+                  {{else}}
+                    {{t 'edit_rating'}}
+                  {{/if}}
```

Comment:
If the survey response is offered but not expired, we should call `{{t 'add_rating'}}`.
If the survey response is offered and expired, I don't know what to show. Do the designs cover this edge case?

Reply:
We don't have designs for viewing a response of an expired survey.
The current implementation shows a message informing end-users that their survey has expired: 
<img width="831" alt="Screenshot 2024-01-29 at 14 15 04" src="https://github.com/zendesk/copenhagen_theme/assets/17469404/c59d6b21-809c-4169-8029-a38e7e45b507">

Reply:
We do have them. We skip showing the whole `rating` detail

![Screenshot 2024-01-29 at 14 18 33](https://github.com/zendesk/copenhagen_theme/assets/4209616/1b0ef4f4-1e8d-4875-bc61-acc24772338b)